### PR TITLE
Fixes the DV Evac shuttles

### LINKED ISSUE #1179: Evac shuttles need announcement screens
## Description
Maps missing screens

- [x] NTES-Propeller 
- [x] NTES-Lox
- [x] NTES-Delta

Should be 1 or 2 in common area depending on size, 1 in engie room, and 1 in bridge. Additional can be added if there are other rooms (sec, med, etc)

### LINKED ISSUE #1689: Shipyard shuttles need Warp Points
## Description
- ~~Shuttles need to be updated with `directional fans` replacing `tiny fans`~~
- Shipyard shuttles need the "Shuttle" `warp point` added
- ~~See also: [#1179](https://github.com/DeltaV-Station/Delta-v/issues/1179)~~

**Additional context**
Easy money. Just remember to `savegrid` **Do not** `savemap`

### PR DESCRIPTION
**Pull request by therealDLondon**
## About the PR
Swaps tiny fans for directional fans on all DV evac shuttles.
And adds screens to others
Fixes #1179 and partially fixes PR-1689 

## Why / Balance
Fan go woosh

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
No sorry

### CODE REVIEW

File: Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml [hunk 1]
```diff
@@ -2804,11 +2794,15 @@ entities:
   - uid: 555
     components:
     - type: Transform
+      anchored: False
```
Comment (IamVelcroboy) [inline Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml]: Looks like the Lox has a duplicate pipe here

Comment (therealDLondon) [reply Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml]: Thats nothing I had to do with, but ill hunt it down

File: Resources/Maps/Shuttles/DeltaV/NTES_UCLB.yml [hunk 2]
```diff
@@ -5365,11 +5369,15 @@ entities:
   - uid: 1180
     components:
     - type: Transform
+      anchored: False
```
Comment (IamVelcroboy) [inline Resources/Maps/Shuttles/DeltaV/NTES_UCLB.yml]: And another one

Comment (therealDLondon) [reply Resources/Maps/Shuttles/DeltaV/NTES_UCLB.yml]: Hey wait a second thats YOUR shuttle! >:3

File: Resources/Maps/Shuttles/DeltaV/NTES_Right.yml [hunk 3]
```diff
@@ -7571,7 +7444,10 @@ entities:
       pos: -7.5,-18.5
       parent: 1
     - type: RandomSpawner
+      prototypes: []
```
Comment (IamVelcroboy) [inline Resources/Maps/Shuttles/DeltaV/NTES_Right.yml]: Can you remap this salvage canister spawner whole your doing this too, please. I don't know why all this extra stuff is on it but I don't like it.

Comment (therealDLondon) [reply Resources/Maps/Shuttles/DeltaV/NTES_Right.yml]: Okidoki

File: Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Maps/Shuttles/DeltaV/NTES_Lox.yml]: I know, just figure while you're in here...thanks 😁

**Review by IamVelcroboy — APPROVED** [review]

**Pull request merged**